### ISSUE TITLE
gemeentes worden op enige schaal gemerged

### ISSUE THREAD

**Issue by Gijs-Koot** [Gijs-Koot/gemeente-deler#8]
Ik zie dat sinds 2002 ongeveer 1 op 5 gemeentes is opgegaan in andere gemeentes. Als we MVP zouden zien als alleen data gebruiken van 2010 en recenter, dan is het nog steeds ongeveer 1 op 10 gemeentes die is opgegaan.

We kunnen MVP zien als dat er dan maar null teruggegeven moet worden voor een gemeente die niet meer bestaat. Maar bijv. Rotterdam, waar over tijd meerdere gemeentes in zijn opgegaan, is dus qua grondgebied uitgebreid. Misschien is het voor zo'n gemeente zinvoller als dan in een tijdreeks bij de eerdere cijfers van voor een fusie de data van de te fuseren gemeente wordt opgeteld.... mmm... moeten we iig op 1 of andere manier communiceren deze shit, helemaal mooi zou zijn beide teruggeven. Dat je van bijv. Rotterdam: per jaar het cijfer ziet van wat t toen was met en zonder aangenomen buurtgemeentes... klinkt niet meer als MVP

**Comment by Gijs-Koot** [comment]
Mijn idee is dat datasets, de statistieken, ook van een jaartal voorzien worden. 
1. Je kiest een gemeente, uit alle keuzes inclusief de historische, mochten we die tegenkomen in een dataset. 2. Als vergelijkingscategorieen kun je alleen kiezen uit de data die voor deze gemeente beschikbaar is, bijv. bevolking 2008, want in 2009 bestaat hij niet meer. 3. Als vergelijking krijg je vervolgens alleen gemeentes die ook deze cijfers hebben, dus een bevolkingsaantal in 2008. Als regels in de tabel zie je ook alleen datasets die voor de gekozen gemeente beschikbaar zijn. Als je iets kiest waarop je niet hebt gezocht, kan het zijn dat je 'vergelijkbare' gemeentes velden missen. Die lege velden zie je dan gewoon.

**Comment by WHPDon** [comment]
Dat klinkt als een goede oplossing.

Vraag: Klopt het dat dit ook jouw uitgangspunt is voor interface -> selectie van gemeente en dan datasets, en vervolgens alle beschikbare jaren teruggeven (met hier en daar lege velden)

**Comment by Gijs-Koot** [comment]
Ja, ik dacht dat als we meer datasets ingeladen hebben, dat we dan een soort selectiemechanisme moeten hebben voor welke regels in de tabel komen. Daarin kunnen dan lege velden ontstaan.
